pkgs/applications/editors/kakoune/plugins.nix

```diff
@@ -0,0 +1,5 @@
+{ parinfer-rust }:
+
+{
+    inherit parinfer-rust;
```

Comment:
```suggestion
  inherit parinfer-rust;
```

Reply:
Amended the commit and force-pushed for this.  Thank you!